### PR TITLE
T364134 New Media Viewer Experience

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator: https://phabricator.wikimedia.org/T364134
Demo: https://wikimedia.github.io/wikipedia-preview/T364134/articles/english.html

### CODE REVIEW

**Review by stephanebisson — DISMISSED** [review]
If you toggle the network to "slow 3G", the image description appears first, right below the image loading state in the middle of the page and then shifts down when the image is loaded. I would recommend only showing the image description when the image is loaded to avoid the shift.

**Review by stephanebisson — CHANGES_REQUESTED** [review]

File: src/gallery/slider.js [hunk 1]
```diff
@@ -54,60 +54,16 @@ const renderImageSlider = ( givenLang, givenDir, container, images = [], selecte
...
-		<div class="${ prefixClassname }-item-caption">
-			${ isCaptionExpandable() ? `<div class="${ prefixClassname }-item-caption-expand-cue"></div>` : '' }
-			${ description ? `<div class="${ prefixClassname }-item-caption-text"><bdi>${ description }</bdi></div>` : '' }
-		</div>
 		<div class="${ prefixClassname }-item-attribution">
 			<div class="${ prefixClassname }-item-attribution-info">
-				${ getLicenseInfo( mediaInfo.license ) }
-				${ author ? `<bdi class="${ prefixClassname }-item-attribution-info-author">${ author }</bdi>` : '' }
+				<bdi>${ author } (${ mediaInfo.license })</bdi>
+				<a href="${ link }" class="${ prefixClassname }-item-attribution-info-link" target="_blank">Learn more ${ getLinkIconSvg() }</a>
```
[the remark this answers is not in the document]
Comment (hueitan) [reply src/gallery/slider.js]: Done.

**Review by stephanebisson — CHANGES_REQUESTED** [review]
When hovering over the icon next to "learn more" I get the following error in the console:
"Uncaught TypeError: e.target.className.indexOf is not a function"

**Comment by hueitan** [comment]
> When hovering over the icon next to "learn more" I get the following error in the console: "Uncaught TypeError: e.target.className.indexOf is not a function"

Good catch, fixed now.

**Review by stephanebisson — APPROVED** [review]

File: style/gallery.less [hunk 2]
```diff
@@ -148,9 +166,14 @@
 					margin: 16px;
 					font-size: 12px;
 					color: var( --gallery-primary-text-color );
-					overflow: hidden;
-					text-overflow: ellipsis;
-					white-space: nowrap;
+					
+					bdi {
```
Comment (stephanebisson) [inline style/gallery.less:170]: I don't think bdi should be used for styling. There must be a div or span with a meaningful class nearby.

**Pull request merged**